src/resources/views/auth/list.blade.php

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="{{ str_replace('_', '-', app()->getLocale()) }}">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <title>クイズ一覧</title>
+    </head>
+    <body>
+      @extends('layouts.app')
+
+      @section('content')
+        @foreach ($links as $link)
+        <div style="text-align: center">
```

Comment:
style属性は基本使わないほうが良いです。
癖になってstyle属性ばかり使うと修正しづらいコードが量産されます
```suggestion
        <div style="text-align: center">
```